features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -63,8 +66,35 @@ data class SessionReplayConfiguration internal constructor(
          * @see SessionReplayPrivacy.MASK
          * @see SessionReplayPrivacy.MASK_USER_INPUT
          */
+        @Deprecated(
+            message = "This method is deprecated and will be removed in future versions. " +
+                "Use the new fine grained masking apis instead: " +
+                "setImagePrivacy, setTouchPrivacy, setTextAndInputPrivacy."
```

Comment:
minor: you can use [setImagePrivacy], [setTouchPrivacy], [setTextAndInputPrivacy] to link the functions in documentation.